src/sentry/static/sentry/app/components/sampleEventBanner.jsx

```diff
@@ -46,14 +51,20 @@ const SampleEventBanner = createReactClass({
     // if no projects - redirect back to onboarding flow
     if (organization.projects.length > 0) {
       let project = organization.projects.pop();
-      url = `/${organization.slug}/${project.slug}/getting-started/${project.platform}`;
+      url = `/onboarding/${organization.slug}/${project.slug}/configure/${project.platform}`;
     } else {
       url = `/onboarding/${organization.slug}`;
     }
 
     return url;
   },
 
+  handleClick() {
```

Comment:
This could be named better- should really be called `recordAnalytics` - I can update when I rebase